TusStorageHandler/tus_server.py

```diff
@@ -0,0 +1,80 @@
+"""TusServer Module defines the TusServer class for handling files."""
+
+import os
+
+from minio.error import S3Error
+
+
+class TusServer:
+  """Initializes a new instance of the TusServer class.
+
+  Args:
+      minio_client (Minio): The Minio client object used for file operations.
+      bucket_name (str): The name of the Minio bucket to operate on.
+  """
+
+  def __init__(self, minio_client, bucket_name):
+    """Initializes a new instance of the TusServer class.
+
+    Args:
+        minio_client (Minio): The Minio client object used for file operations.
+        bucket_name (str): The name of the Minio bucket to operate on.
+
+    """
+    self.minio_client = minio_client
+    self.bucket_name = bucket_name
+
+  def get_file_path(self, upload_id):
+    """Returns the file path for the given upload_id within the 'uploads' directory.
+
+    Args:
+        upload_id (str): Unique identifier for the upload.
+
+    Returns:
+        str: File path within the 'uploads' directory corresponding to the upload_id.
+
```

Comment:
whats up with spaces after pydocstring, ruff won't pick these up ig as they don't violate with googles pydocstring guide (or I think so), but either be consistent and leave spaces in all or dont in any, I prefer latter :).